docker-compose.yml

```diff
@@ -57,6 +57,13 @@ services:
     depends_on:
       - varnish
 
+  memcached:
+    build:
+      context: ./memcached/1.4/
+    image: quay.io/continuouspipe/memcached1.4:v1.0
```

Comment:
`quay.io/continuouspipe/memcached-1.4:v1.0` would prob be more preferable for me, but I don't think we should change it for just one image